src/variant.ts

```diff
@@ -32,18 +32,14 @@ export interface Variant {
   name: string;
   enabled: boolean;
   payload?: Payload;
+  feature_enabled?: boolean;
```

Comment:
Do we need this one to be optional here? 🤔 Would it break anything if we made it required?

Reply:
We can make it required in next version. I'd like to avoid breaking "fallback variant" interface